seeds/colors.js

```diff
@@ -0,0 +1,16 @@
+const colors = require('../colors');
+
+const hexColors = colors.map(color => ({
+  hex_code: color
+}));
+
```

Comment:
I'm a little confused about the code in this file. Let's review it in person, though. 
Appreciate ya getting this done.